### ISSUE TITLE
[Feature] JSON export of recurring_expenses

### ISSUE THREAD

**Issue by iokiwi** [iokiwi/moneyapp#32]
Create an "Export" dropdown button on the https;//localhost:8000/recurring_expenses/ page

Bootstrap drop down button: https://getbootstrap.com/docs/5.3/components/button-group/#nesting

Add an option drop down option for "JSON"

When clicked, the user should get an export of their recurring expenses including the following fields for each ID

 * id
 * Name
 * Period
 * Currency
 * Amount
 * Amount NZD

Note that until #30 is implemented, you can create your new recurring expense entries by going to this url https;//localhost:8000/recurring_expenses/new

**Comment by iokiwi** [comment]
@abhinna1 hey stargazer! I notice you starred this repo. I wondered if you were considering contributing as part of hacktober fest. 
I think this issue would be a great place to start with your first PR. I am happy to help provide advice/mentoship

**Comment by abhinna1** [comment]
@iokiwi Sure, I could definitely look into it! 😃

**Comment by abhinna1** [comment]
Hi @iokiwi , I had a small confusion about the description of this issue. So as per my understanding, we want an "Export" drop-down (maybe alongside the "add new" button). And the dropdown consists of the columns the user wants to export? And upon clicking export, maybe we want the data to be exported into a downloadable csv file (Maybe this could be registered as a different issue but not a problem if we do it here either).

**Comment by abhinna1** [comment]
@iokiwi So I did get what you meant. I have created an export drop-down with a JSON option in the template.
For now, I've handled the download login in the client-side with vanilla JavaScript. Do we want to achieve the export login from the client-side or from the server side maybe creating a separate view?
We can attempt anything you'd like.
**Export File:**
![image](https://github.com/iokiwi/moneyapp/assets/24387757/b4e10ec9-282d-4a52-b291-6dc96d6e7ec3)
